squidDrone/src/mcu_config/mcu_settings.c

```diff
@@ -1,7 +1,9 @@
 #include "mcu_settings.h"
 
 void Error_Handler(){
-
+    for(;;){
```

Comment:
Error_Handler implements the behaviour in case of a mcu config error.
Error_Handler is generated by the startp code generator.
By default the implementation is empty, but for now I would like to not ignore mcu config errors and implemented a system halt. 
By doing this the code is debuggable and the stack is still intact.